### PR TITLE
Contract data

### PR DESCRIPTION
**Pull request by hone1er**
### What changes were made?

- Saves the names of attached files to the bountyState/S3/contract object
- Add additional error handling for posting werk files

### Is there any background info about the change?

- provides errors if file upload fails
- all data is being sent to S3 bucket and appropriate data sent to contract
TODO:
 - allow only ETH for payment(?)
 - convert ETH amount to wei(?)
 - error handling/check for completeness

### Screenshots or Gifs✨

Screen Shot 2022-07-14 at 11 40 35 AM
Screen Shot 2022-07-14 at 11 39 59 AM
Screen Shot 2022-07-14 at 11 38 03 AM

### Any new dependencies? Why were they added?

na

### Shortcut Link

https://app.shortcut.com/twali/story/1001/send-all-required-data-from-sow-builder-to-s3-bucket-and-smart-contract

### CODE REVIEW

**Review by nagmak — APPROVED** [review]
Looks good to me, however if you have a moment @NickGonzalez04 pls confirm that this looks good on your end too

File: components/HeaderNav/HeaderNav.tsx [hunk 1]
```diff
@@ -116,9 +124,9 @@ const HeaderNav = (props) => {
                     letterSpacing={"0.06em"}
                     textTransform={"uppercase"}
                     padding="4px 8px"
-                    isTruncated
+                    noOfLines={1}
```
Comment (NickGonzalez04) [inline components/HeaderNav/HeaderNav.tsx:127]: You meant to put this 👀 ?

**Review by NickGonzalez04 — APPROVED** [review]
Looks good! I would add to your 'TODO' to make sure we set the production bucket before the feature goes into staging!

**Comment by NickGonzalez04** [comment]
@hone1er you can merge this into your branch. Just left a comment but you are what they say `gucci`

**Pull request merged**